src/utilities/richTextField.js

```diff
@@ -8,10 +8,7 @@ const Storyblok = new StoryblokClient({});
 
 function createMarkup(storyblokHTML) {
   // Run sanitizer and renderer.
-  let markup = sanitizeHtml(Storyblok.richTextResolver.render(storyblokHTML), {
-    allowedTags: false, // This allows extra HTML tags to be rendered, e.g., H2, img
-    allowedAttributes: false, // This allows custom class names
-  });
```

Comment:
I'm concerned about this change as I don't know which tags and attributes we currently have in Storyblok. Removing these configuration items sets sanitizeHTML to the defaults, which is more restrictive than the previous settings. For example, the default HTML tags of sanitizeHTML does not include the `img` tag. I'm 99.9% sure we allow for images in WYSIWYG somewhere in our blok library. I also don't know which attributes that the richtext (wysiwyg) editor in Storyblok allows for and if they match. If you want to go down this path to fix the warning error you will need to go into our Storyblok space and account for all the HTML tags and attributes that are allowed, compare them to the defaults of sanitizeHTML, and add the missing ones back in.